continuous_integration.md

```diff
@@ -25,81 +25,37 @@ to re-run the test suite for all target platforms on every single code
 base change to the shoulders of the developers.
 
 In order to address this problem, a
-[Jenkins](http://jenkins-ci.org/)-based CI infrastructure was created
-during the [NEST Google Summer of Code
-2011](http://www.google-melange.com/gsoc/project/google/gsoc2011/zaytsev/17001)
-project, which would allow for regular testing of changes that are
+[Travis](http://travis-ci.org/)-Continuous Integration (CI) service has been setup,
+which would allow for regular testing of changes that are
 getting into the tree and timely reporting of identified
 problems. This way, issues will be discovered earlier and the amount
 of efforts to fix them will be significantly decreased (hopefully).
-Continuous Integration at NEST Initiative
+Continuous Integration at NEST Simulator
 
 The current CI implementation is now available at the following URL:
 
-    https://qa.nest-initiative.org/
+    https://travis-ci.org/nest/nest-simulator
 
-The site is secured with a Class 1 SSL certificate issued by StartCom Ltd.
+Travis CI service has been integrated with the Github repository to automatically run the tests when code is pushed. Github integration is done by adding a simple YAML file to the project root.Travis results will appear in the Github pull requests and the primary log is visible in the Travis interface. 
 Build jobs
 
-The CI system is set up to regularly poll the version control
-repository hosting the main official NEST source tree. Whenever
-changes are detected, the latest source code is downloaded to a build
-executor machine and the following actions are performed:
+The CI system is closely integrated to Github repository. Whenever some changes in the code
+are detected, the latest source code is downloaded to a worker machine which is running on Ubuntu 14.04 LTS(Trusty) and the following actions are performed:
 
     Bootstrap the build system
     Build and install NEST
     Run the test suite
-    Bootstrap MyModule
-    Build and install MyModule
+    Uploading the build artifacts to Amazon S3 (Simple Storage Service)
 
-If any of these steps fails (returns a non-zero exit code), the build
-is marked as failed and a notification is sent to the developers
-mailing list.
+If any of the first three steps fails (returns a non-zero exit code), the build
+is marked as failed and a notification is sent to the person who initiated the build.
 
-Additionally, a number of metrics are collected from the build log to
-obtain a cumulative project "health" indicators:
+Due to security reasons, Travis doesn't allow uploading of artifacts to S3 for the pull requests.
```

Comment:
From a pull request, the secure values (for e.g.: the encrypted access keys for S3) couldn't be extracted properly and thus the artifacts can't be uploaded to S3. Secure access keys are set for the particular repository (in our case, nest/nest-simulator). Because of this reason, artifacts will be uploaded when we merge the commit to the upstream. 
May be I could add this information along with line 53.